console/www/tooltips.json

```diff
@@ -0,0 +1,63 @@
+{
+    "address": {
+        "anycast": {
+            "label": "A scalable 'direct' address for sending messages to one consumer",
+            "plan": {
+                "inmemory": {
+                    "label": "Configures router network with anycast address"
+                }
+            },
+            "tooltip": "The `anycast` address type is a scalable 'direct' address for sending messages to one consumer. Messages sent to an `anycast` address is not stored but forwarded directly to the consumer. This makes this address type ideal for request-reply (RPC) uses or even work distribution. This is the cheapest address type as it doesn't require any persistence."
+        },
+        "multicast": {
+            "label": "A scalable 'direct' address for sending messages to multiple consumers",
+            "plan": {
+                "inmemory": {
+                    "label": "Configures router network with multicast address"
+                }
+            },
+            "tooltip": "The `broadcast` address type is a scalable 'direct' address for sending messages to multiple consumers. Messages sent to a `broadcast` address are forwarded to all consumers receiving on that address. It is important to note that only pre-settled messages can be sent to broadcast addresses, as message acknowledgements from consumers are not propagated to producers."
+        },
+        "name": {
+            "external": "https://github.com/EnMasseProject/enmasse/pull/44",
+            "label": "Enter the address name",
+            "tooltip": "The <b>address name</b> is a unique string to which messages can be sent and received."
+        },
+        "queue": {
+            "label": "A store and forward queue",
+            "plan": {
+                "inmemory": {
+                    "label": "In memory only",
+                    "tooltip": "Creates a standalone broker cluster for queues. Messages are not persisted on stable storage."
+                },
+                "persisted": {
+                    "label": "Saved to storage",
+                    "tooltip": "Creates a standalone broker cluster for queues. Messages are persisted on stable storage."
```

Comment:
Is this the file that is generated by the script after looking at the docs? I assume this file would essentially be reused for the address controller /v1/schema as well, and the names (at least tooltip) has a very UI-specific meaning. Does it make sense to rename label -> shortDescription, tooltip -> longDescription or someting like that?

Reply:
Yes, this is the file that is generated by the script from parsing the docs. 
And yes, shortDescription and longDescription sound like better names. I'll make that change.